packages/api/models/nomination.js

```diff
@@ -44,12 +44,26 @@ module.exports = (sequelize, DataTypes) => {
       amountRequestedCents: DataTypes.INTEGER,
       amountGrantedCents: DataTypes.INTEGER,
       attachmentsDestination: DataTypes.STRING,
+      publicEmailDomain: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false,
+      },
     },
     {
+      hooks: {
+        beforeCreate: (nomination, option) => {
+          const negEmails = ['gmail.com', 'aol.com', 'outlook.com', 'zoho.com', 'mail.com', 'yahoo.com', 'protonmail.com', 'icloud.com'];
```

Comment:
Nit: I might move these up toward the top, for easy editing in the future (and to prevent redeclaring it every time we create a nomination).